File: leetcode/csharp/2938.separate-black-and-white-balls.cs
public class Solution {
    public long MinimumSteps(string s) {
        long ans = 0, ones = 0;
        for (int i=0; i < s.Length;i++) {
            if (s[i] == '1') {
                ones++;
            } else {
                ans += ones;
            }
        }
        return ans;
    }
}

File: leetcode/cpp/2938.separate-black-and-white-balls.cpp
class Solution {
public:
    long long minimumSteps(string s) {
        long ans = 0, ones = 0;
        for (const char c: s) {
            switch (c) {
                case '1':
                    ones++;
                    break;
                default:
                    ans += ones;
                    break;
            }
        }
        return ans;
    }
};

